src/foam/comics/v2/DAOCreateView.js

```diff
@@ -86,15 +97,22 @@ foam.CLASS({
       },
       code: function() {
         var cData = this.data;
+        var message = this.SUCCESS_CREATED_MESSAGE;
 
         if ( foam.nanos.auth.LifecycleAware.isInstance(cData) ) {
           cData = cData.clone();
           cData.lifecycleState = foam.nanos.auth.LifecycleState.PENDING;
```

Comment:
What happens if the user creates the data through DIG instead? Then do they avoid going through lifecycle?

Reply:
The default lifecycleState property value for User is ACTIVE and so they should be able to skip through the approval process as I just accounted for it in the approvableAwareDAO decorator.

Otherwise if you wanted to change the default value to PENDING, we would have to make a DIG user create be able to set LifecycleState to ACTIVE before putting it into the DAO.